spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestCopyOnWriteUpdate.java

```diff
@@ -78,6 +81,7 @@ public synchronized void testUpdateWithConcurrentTableRefresh() throws Exception
         tableName, UPDATE_ISOLATION_LEVEL, "snapshot");
 
     sql("INSERT INTO TABLE %s VALUES (1, 'hr')", tableName);
+    createBranch();
```

Comment:
Similar to above, you may want to check this test. Appends were updated, but not updates.

Reply:
missed the change of the line later to use `commitTarget`, updated

Reply:
updated!